### ISSUE TITLE
Terminal buffer reading using screen reader

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#270]
When using command control + sdta upwards to read the terminal buffer, it is not possible to read the entire log.

Example when running a build of a java project with the mvn clean install command, thousands of log lines are produced when pressing control + up arrow I do not navigate through this entire report.

I think this command should only be pressed once and then it could be possible to navigate only using arrows and, advance or rewind a page when using pageup and pagedown and when pressing control plus down arrow it could exit the buffer area as it already does today.

I use the latest version of Nvda as well as the latest version of Vscode.

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold
